origin/globus.go

```diff
@@ -63,21 +66,29 @@ const (
 	GlobusActivated = "Activated"
 )
 
-const GlobusTokenFileExt = ".tok" // File extension for caching Globus access token
+const GlobusTokenFileExt = ".tok"                  // File extension for caching Globus access token
+const GlobusTransferTokenFileExt = ".transfer.tok" // File extension for caching Globus transfer token
```

Comment:
I don't see that these are used outside the `origin` package -- do they need to be exported? If not, can you lowercase them?